Lessened the pain of players and medical doctors by buffing some of the worst c2 chemicals to a slightly better state

MacBlaze1 opened the pull request:
## About The Pull Request

PR Theme: https://www.youtube.com/watch?v=g8Adf53SkTY

Changes Aiuri to have decreased eye damage because these are still supposed to be shit chems and C2 ones. 
Changes instabitaluri to be have a lessened toxin conversion rate to make it also more viable to be used by doctors in ghetto chem.

## Why It's Good For The Game

I have spoken to numerous people who believe C2 chems are very very bad, worse than they should be infact. I am not here to fix that today. I am here to ease the pain of these chems and breathe a little life into them by making them better while keeping the downsides they have. These are not supposed to be amazing chems, and they arent. But right now these chems are not very good, and should be improved just a bit so 1/10 players might use them. Seeing as how widespread these are, and how they have replaced the other chems in medkits, sleepers, actual purpose, there should be some incentive to use them.

## Changelog

:cl:
tweak: Changed the eye damage of aiuri to .25 from 1, does not change healing.
tweak: Changed toxin conversion of instabitaluri to 66% from 75%, does not buff healing.
balance: Changed eye damage giving rate of aiuri and toxin conversion of  instabitaluri 
/:cl:

MacBlaze1 committed:
Update cat2_medicine_reagents.dm

ExcessiveUseOfCobblestone reviewed (changes requested):
Aiuri - Drop the eye damage instead (Why different organs have different metrics is idk)

Synthflesh - I'll accept 2/3rds, I don't want an entire damage type nulled out.

MacBlaze1 committed:
Cobby changes

Reverted burn healing to 0.5, removed eye damage from aiuri. Changed toxin damage conversion on instabwhatever to 66%

ExcessiveUseOfCobblestone commented:
Sorry when I meant drop the eye damage I meant lessen it to something sane, not remove it entirely.

ExcessiveUseOfCobblestone commented:
I'll early merge since this is hitting the default burn chem.

MacBlaze1 committed:
Update cat2_medicine_reagents.dm
